src/mlpack/core/optimizers/cmaes/cmaes.hpp

```diff
@@ -166,8 +166,7 @@ class CMAES
 /**
  * Convenient typedef for CMAES approximation.
  */
-template <typename SelectionPolicyType = RandomSelection>
-using ApproxCMAES = CMAES<SelectionPolicyType>;
+typedef CMAES<RandomSelection> ApproxCMAES;
```

Comment:
What I really like about ``using`` is that you can use ``ApproxCMAES``, ``ApproxCMAES<>`` or ``ApproxCMAES<..>``.

Reply:
Ah, I had not thought about that.  Reverted.